### ISSUE TITLE
node-wcag tests failing

### ISSUE THREAD

**Issue by ascott1** [cfpb/capital-framework#183]
The node-wcag tests seem to be failing for many (if not all) cf components. Here's the output:

<img width="936" alt="screen shot 2015-07-31 at 9 43 33 am" src="https://cloud.githubusercontent.com/assets/212533/9008498/c2b37556-3768-11e5-8d9e-cbe3b811d805.png">

Since the errors aren't specified, it's hard to diagnose what the problem is. @contolini any ideas?

**Comment by contolini** [comment]
With `cf-buttons`, it's because the demo page has buttons that include [links without content](https://github.com/cfpb/cf-buttons/blob/gh-pages/demo/index.html#L311-L313). This violates WCAG-AA guidelines.

The proper way to do compound buttons would be to add visually hidden text:

```html
<a href="#" class="btn btn__grouped-last btn__compound-action">
    <span class="u-visually-hidden">Dropdown</span>
    <span class="cf-icon cf-icon-down btn__grouped-last"></span>
</a>
```

**Comment by anselmbradford** [comment]
Testing file still exists in https://github.com/cfpb/capital-framework/blob/canary/test/accessibility/wcag.js
But the command to run it was removed in https://github.com/cfpb/capital-framework/commit/927006327fb78a098e93474560c01972dc02f22b

**Comment by ascott1** [comment]
Closing this issue. Thanks @anselmbradford!
